addons/account/views/report_invoice.xml

```diff
@@ -133,7 +133,7 @@
 
                                     <tr t-att-class="'bg-200 fw-bold o_line_section' if line.display_type == 'line_section' else 'fst-italic o_line_note' if line.display_type == 'line_note' else ''">
                                         <t t-if="line.display_type == 'product'" name="account_invoice_line_accountable">
-                                            <td name="account_invoice_line_name"><span t-field="line.name" t-options="{'widget': 'text'}">Bacon Burger</span></td>
+                                            <td name="account_invoice_line_name"><span t-if="line.name" t-field="line.name" t-options="{'widget': 'text'}">Bacon Burger</span></td>
```

Comment:
Do you know why `Bacon Burger` is still displayed in Studio? :exploding_head:

Reply:
Ok, I think it is because we consider that `t-if` is always true, but `t-elif` and `t-else` are always false? :thinking: